src/v2/Components/RecentlyViewed/V3RecentlyViewed.tsx

```diff
@@ -0,0 +1,106 @@
+import { ContextModule } from "@artsy/cohesion"
+import { Shelf, Text } from "@artsy/palette"
+import { V3RecentlyViewed_me } from "v2/__generated__/V3RecentlyViewed_me.graphql"
+import { V3RecentlyViewedQuery } from "v2/__generated__/V3RecentlyViewedQuery.graphql"
+import { SystemContext } from "v2/Artsy"
+import { useTracking } from "v2/Artsy/Analytics"
+import * as Schema from "v2/Artsy/Analytics/Schema"
+import { SystemQueryRenderer as QueryRenderer } from "v2/Artsy/Relay/SystemQueryRenderer"
+import React, { useContext } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { ShelfArtworkFragmentContainer } from "../Artwork/ShelfArtwork"
+import { V3RecentlyViewedPlaceholder } from "./V3RecentlyViewedPlaceholder"
+
+export interface V3RecentlyViewedProps {
+  me: V3RecentlyViewed_me
+}
+
+export const V3RecentlyViewed: React.FC<V3RecentlyViewedProps> = ({ me }) => {
```

Comment:
Can we update this name? `RecentlyViewedV3` follows other idioms

Reply:
Ah, yeah sure